torch/_dynamo/variables/lists.py

```diff
@@ -482,6 +482,28 @@ def call_hasattr(self, tx: "InstructionTranslator", name: str) -> "VariableTrack
         return variables.ConstantVariable.create(hasattr([], name))
 
 
+class FxImmutableListVariable(ListVariable):
```

Comment:
Can you add a test to see if this is actually immutable under Dynamo? Would not be good if one could mutate it under dynamo but not eager

Reply:
Done